README.md

```diff
@@ -38,12 +38,17 @@ $ yarn
 ## Development
 
 ```sh
+# Compile the source files (only needed the first time)
+$ yarn build
+
 # Start the app
 $ yarn watch
 ```
 
 The app is running on `http://localhost:8080`. Visit [http://localhost:8080/se/new-member/](http://localhost:8080/se/new-member/) to view the first screen of the onboarding flow.
 
+> The port might differ on your machine. Check the logs for this message: `Booting server on 8040 👢`.
+
```

Comment:
Ah good catch! 🙌 The port was changed at some point and it hasn't been updated here, should be 8040 all the time 😄
```suggestion
The app is running on `http://localhost:8040`. Visit [http://localhost:8040/se/new-member/](http://localhost:8080/se/new-member/) to view the first screen of the onboarding flow.

```

Reply:
Thanks - updated now and removed the callout section